.gitpod.Dockerfile

```diff
@@ -0,0 +1,52 @@
+FROM eu.gcr.io/gitpod-core-dev/dev/dev-environment:cw-bump-leeway.29
+
+# Install Xvfb, JavaFX-helpers, Openbox window manager and emacs GTK
+RUN sudo install-packages xvfb x11vnc xterm openjfx libopenjfx-java openbox \
+    emacs-gtk
+
+# Install Cypress dependencies.
+RUN sudo apt-get update \
```

Comment:
Please use the same approach `sudo install-packages`

Reply:
@aledbf thanks for taking the time to look at this 🙏 

🥷🏻 I took this part directly from https://github.com/mikenikles/cypress-on-gitpod/blob/master/.gitpod.Dockerfile but I don't expect it to stay like this.

🔮 I'm guessing that if the PR ends up getting accepted, all the packages and configurations in this file should be directly included 📦 in the docker image (e.g. `eu.gcr.io/gitpod-core-dev/dev/dev-environment:cw-bump-leeway.29`) instead.

🎯 This is the reason why I have a bullet point in the TODO (see the description of this PR) about getting someone from Gitpod to help me better understand, and implement, this update of the usual docker image:

> Ask for guidance about properly updating the docker image and the structural changes at large (I basically hacked together things by @mikenikles in a way that's temporary to say the least)